devtools/maven/src/main/java/io/quarkus/maven/RunMojo.java

```diff
@@ -0,0 +1,112 @@
+package io.quarkus.maven;
+
+import java.nio.file.Path;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+
+import io.quarkus.bootstrap.app.AugmentAction;
+import io.quarkus.bootstrap.app.CuratedApplication;
+import io.quarkus.deployment.run.RunCommandHandler;
+import io.quarkus.deployment.run.RunCommandLaunchResultBuildItem;
+
+@Mojo(name = "run")
+public class RunMojo extends QuarkusBootstrapMojo {
```

Comment:
`mvn quarkus:run` fails with:`Error: Unable to access jarfile /Users/manderse/temp/myj/target/quarkus-app/quarkus-run.jar`

can we make it require project build first or at least tell the user to have it built?

Reply:
Wouldn't this require changes to pom.xml